src/pages/Calculator.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-len */
```

Comment:
- [ ] Just as I suggested above, you are not meant to use any linters disable rule on your code base, unless otherwise stated. The linters are there to make sure your code follows the best practices, I'll suggest you remove these disable rules and fix the issues. :+1: :smile: :small_airplane: